app/src/components/main/grid/gridBody/gridRow/gridRow.jsx

```diff
@@ -19,6 +19,8 @@ export class GridRow extends Component {
     selectable: PropTypes.bool,
     selectedItems: PropTypes.arrayOf(PropTypes.object),
     onToggleSelection: PropTypes.func,
+    bulkSelection: PropTypes.bool,
```

Comment:
Is it really needed? You can just not pass the corresponding handler if bulk selection not needed

Reply:
Fixed